File: ulohy/21-brigady/reseni/main.cpp
#include <iostream>
#include <vector>

int main() {
    int m;
    std::cin >> m;

    std::vector<int> hoursWorked(m);

    for (int i = 0; i < m; i++) {
        std::cin >> hoursWorked[i];
    }


    int salary = 0;
    for (int i = 0; i < m; i++) {
        int hourlyRate;
        std::cin >> hourlyRate;
        salary += hoursWorked[i] * hourlyRate;
    }

    std::cout << salary << std::endl;

    return 0;
}

File: ulohy/21-brigady/reseni/main.py
def read_int() -> int:
    return int(input())


if __name__ == "__main__":
    m = read_int()
    input()

    hours_worked = []
    for i in range(m):
        hours_worked.append(read_int())

    input()

    salary = 0
    for i in range(m):
        hourly_rate = read_int()
        salary += hours_worked[i] * hourly_rate

    print(salary)
